docs/api-docs/storefront/graphql/graphql-api-examples.mdx

```diff
@@ -4,12 +4,47 @@ keywords: storefront, graphql
 ---
 # GraphQL Storefront API Example Queries
 
-Below are example GraphQL queries for use with the BigCommerce GraphQL Storefront API. The purpose of these examples is to assist developers in getting familiar with the API. For a general overview of it's usage and capabilities, see [GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview).
+Below are example GraphQL queries for use with the BigCommerce GraphQL Storefront API. The purpose of these examples is to assist developers in getting familiar with the API. For a general overview of its usage and capabilities, see [GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview).
 
-<Callout type="warning">
+<Callout type="info">
   The GraphQL Storefront API is feature-complete.
 </Callout>
 
+## Configuring the request
+
+To get started, you need a BigCommerce store and a Storefront API token. For more information, see the [Authentication section of the GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview#authentication) or the [Authentication article](/api-docs/getting-started/authentication#bigcommerce-generated-jwts).
+
+For more information on formatting the request payload, see [the query example section of the GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview#querying-within-a-bigcommerce-storefront).
+
+To use this API from a coupled storefront, use the following HTTP configuration:
+
+```js filename="Example query: Same-origin GraphQL Storefront HTTP configuration" showLineNumbers copy
+  fetch('/graphql', {
+    method: 'POST',
+    credentials: 'same-origin',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + {{storefrontToken}}
+    },
+    body: JSON.stringify({
+      query: gqlQueryString
+    })
+  })
+  .then(response => response.json())
+  .then(result => console.log(result))
+  .catch(error => console.error(error));
+```
+
+To use this API server-to-server, use the following HTTP configuration:
+
+```http filename="Server-to-server GraphQL Storefront HTTP configuration" showLineNumbers copy
+  POST https://{{storeDomain}}/graphql
+  Authorization: Bearer {{storefrontOrCustomerImpersonationToken}}
+  X-BC-Customer-ID: 123 # optional once a customer is signed in
+  Accept: application/json
```

Comment:
@slsriehl It looks like `Accept` is not needed, not sure why that is
![no accept header](https://github.com/bigcommerce/dev-docs/assets/96258747/c8052ce2-9930-4972-852c-433f9daeee9e)